_data/simple-icons.json

```diff
@@ -7205,6 +7205,11 @@
             "hex": "FA7343",
             "source": "https://developer.apple.com/develop/"
         },
+        {
+            "title": "Swiggy",
+            "hex": "FC8019",
+            "source": "https://oppositehq.com/portfolio/swiggy-brand-identity-design/"
```

Comment:
Could you update the source to be the [brand's website](https://www.swiggy.com/) - given the SVG is available there - and is likely to better reflect any changes in the future.